### PR TITLE
Add workaround for easyrsa issue

### LINKED ISSUE #528: Random number generator cannot be seeded
While following the Quick Start instructions in the main README, I got the following error while running the `ovpn_initpki` script:
```
Can't load /etc/openvpn/pki/.rnd into RNG
140565960293704:error:2406F079:random number generator:RAND_load_file:Cannot open file:crypto/rand/randfile.c:98:Filename=/etc/openvpn/pki/.rnd
```

I'm not sure if this is the expected behaviour, or if there is a call to `rand` missing in the script.

I figured I'd create an issue in case, feel free to close if this is already known. 

Full log below:

```
$ docker run -v $OVPN_DATA:/etc/openvpn --log-driver=none --rm -it kylemanna/openvpn ovpn_initpki


init-pki complete; you may now create a CA or requests.
Your newly created PKI dir is: /etc/openvpn/pki


Using SSL: openssl OpenSSL 1.1.1c  28 May 2019

Enter New CA Key Passphrase: 
Re-Enter New CA Key Passphrase: 
Generating RSA private key, 2048 bit long modulus (2 primes)
xx
xx
e is 65537 (0x010001)
Can't load /etc/openvpn/pki/.rnd into RNG
140565960293704:error:2406F079:random number generator:RAND_load_file:Cannot open file:crypto/rand/randfile.c:98:Filename=/etc/openvpn/pki/.rnd
```

### PR DESCRIPTION
**Pull request by codebling**
Fixes #528. Easyrsa looks for a `.rnd` file that doesn't exist in `/etc/openvpn/pki`, causing an error to be displayed while building the CA, and possibly making CA certs insecure. 

This PR fixes that by removing the erroneous line in the .cnf prior to building the CA. 

Here is the easy-rsa issue https://github.com/OpenVPN/easy-rsa/issues/261

### CODE REVIEW

**Comment by codebling** [comment]
ping @kylemanna

**Comment by mkozjak** [comment]
@kylemanna - any way to get this in so we don't have to fork it and use our own images?

**Comment by kylemanna** [comment]
What's the behavior without the line? Where is the random data read from?

**Comment by codebling** [comment]
TL;DR upgrade to easy-rsa 3.0.7 and this PR can safely be closed without merging. 

@kylemanna per #528, without this fix, `ovpn_initpki` fails with error
```
Can't load /etc/openvpn/pki/.rnd into RNG
140565960293704:error:2406F079:random number generator:RAND_load_file:Cannot open file:crypto/rand/randfile.c:98:Filename=/etc/openvpn/pki/.rnd
```

...or at least it did on easy-rsa 3.0.6. Took a year and a half to release 3.0.7 but it should finally be fixed there. 

easy-rsa 3.0.7 makes this PR redundant

**Comment by mkozjak** [comment]
@codebling @kylemanna Bear in mind that with 3.0.7 the easyrsa vars file is mandatory and this project does not create it by default, I think. If the vars file isn’t present in the volume, initpki will fail.

**Comment by codebling** [comment]
@mkozjak didn't see that, nice catch. Honestly don't remember how config/vars file works in this project, but sounds like you do.

**Comment by kylemanna** [comment]
Sounds like this is fixed in later versions of EasyRSA, I'd rather fix that then modify the default config.
